pkgs/applications/misc/golden-cheetah/default.nix

```diff
@@ -56,8 +56,7 @@ in mkDerivation rec {
     runHook postInstall
   '';
 
-  # RCC: Error in 'Resources/application.qrc': Cannot find file 'translations/gc_fr.qm'
-  enableParallelBuilding = false;
+  enableParallelBuilding = true;
```

Comment:
Can remove this line completely, already set in setupHook:
https://github.com/NixOS/nixpkgs/blob/323c2ba6b7c64ce942ec33b1e9545c352baa4404/pkgs/development/libraries/qt-5/hooks/qmake-hook.sh#L26-L29